pkg/pillar/kubeapi/kubeapi.go

```diff
@@ -92,6 +90,7 @@ func GetNetClientSet() (*netclientset.Clientset, error) {
 	return nclientset, nil
 }
 
+/*
```

Comment:
Is there no better way to keep this around but not compile it?
And please add a comment about it being removed (I'm guessing you want to keep if it we switch back again in the future.)

Reply:
Yes I will add the comment, yes wanted to keep for reference and switchback if need be.

Reply:
Done and updated commit.